src/librustc_metadata/decoder.rs

```diff
@@ -432,7 +433,7 @@ pub fn get_adt_def<'tcx>(intr: &IdentInterner,
             did: did,
             name: item_name(intr, doc),
             fields: get_variant_fields(intr, cdata, doc, tcx),
-            disr_val: 0,
+            disr_val: ConstInt::Infer(0),
```

Comment:
What's the default type for enum discriminants or the univariant discriminant? `usize`?